clients/cobol-lsp-vscode-extension/src/test/suite/processorgroups.spec.test.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as assert from "assert";
+import * as helper from "./testHelper";
+import * as vscode from "vscode";
+import * as path from "path";
+import * as fs from "fs";
+import { getWorkspacePath } from "./testHelper";
+
+interface PgmConf {
+  program: string;
+  pgroup: string;
+}
+
+function remove_pgm_cfg(pgmConf: PgmConf) {
```

Comment:
Javascript/Typescript convention is to use `camelCase` rather than `snake_case`.